packages/app/screens/landing/LandingScreen.tsx

```diff
@@ -49,8 +50,10 @@ function LandingScreen() {
           data: data.payload
         }); 
       }
-    } catch(error) {
-      console.log(error.message);
+    } catch(error: any) {
+      let message = 'Unknown Error';
+      if (error instanceof Error) message = error.message;
```

Comment:
Put curly brackets around code blocks even if they're only one line to keep consistency with the rest of the codebase.

Reply:
Issue should be resolved in the latest commit.